testcases/OpTestGSBStaticKey.py

```diff
@@ -249,7 +210,9 @@ def collectData(self):
 
     def validateData(self, enable=True):
         # Check the hmc secure boot state
-        hmc_secureboot = self.check_hmc_secureboot_state()
+        sb_utilities = OpSecureBootUtilities(self.cv_HMC, self.hmc_con)
+        hmc_secureboot = sb_utilities.check_hmc_level_secureboot_state()
```

Comment:
if moved to existing class of HMCUtils than this is just a simple call

self.cv_HMC.check_hmc_level_secureboot_state()